src/app.tsx

```diff
@@ -88,6 +91,15 @@ export const App = () => {
                             .filter(promise => promise.status === 'rejected')
                             .map(promise => promise.reason.message);
                     setError(errorMsgs.join(', '));
+                    // Get the node devices in the background since
+                    // they are expensive to get and not important for
+                    // displaying VMs.
+                    nodeDeviceGetAll({ connectionName }).catch(exc => {
+                        addNotification({
+                            text: "Failed to retrieve node devices",
+                            detail: String(exc),
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2371-f4a68508-20251022-105509-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)